modules/platforms/dotnet/Apache.Ignite.Core.Tests/Binary/BinaryDateTimeTest.cs

```diff
@@ -110,6 +111,34 @@ public void TestClassAttributes()
             AssertTimestampField<DateTimeClassAttribute2>((o, d) => o.Value = d, o => o.Value, "Value");
         }
 
+#if NETCOREAPP
```

Comment:
I think this test will work on .NET Framework too (and we should have a test that works on .NET Framework).

Reply:
Fixed.